packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -1084,3 +1085,91 @@ describeCompat("IdCompressor Summaries", "NoCompat", (getTestObjectProvider, com
 		assert(dds2.id.length > 8, "DDS's ID in attached container should not be short");
 	});
 });
+
+const dataStoreCount = 13;
+describeCompat("IdCompressor basic", "NoCompat", (getTestObjectProvider, apis) => {
+	const {
+		dataRuntime: { DataObject, DataObjectFactory },
+		containerRuntime: { ContainerRuntimeFactoryWithDefaultDataStore },
+	} = apis;
+	class TestDataObject extends DataObject {
+		public get _root() {
+			return this.root;
+		}
+
+		public get _context() {
+			return this.context;
+		}
+	}
+
+	const configProvider = createTestConfigProvider();
+	let provider: ITestObjectProvider;
+	const defaultFactory = new DataObjectFactory("T", TestDataObject, [], []);
+
+	const runtimeOptions: IContainerRuntimeOptions = {
+		enableRuntimeIdCompressor: "on",
+	};
+
+	const runtimeFactory = createContainerRuntimeFactoryWithDefaultDataStore(
+		ContainerRuntimeFactoryWithDefaultDataStore,
+		{
+			defaultFactory,
+			registryEntries: [[defaultFactory.type, Promise.resolve(defaultFactory)]],
+			runtimeOptions,
+		},
+	);
+
+	const createContainer = async (): Promise<IContainer> =>
+		provider.createDetachedContainer(runtimeFactory, { configProvider });
+
+	beforeEach("getTestObjectProvider", async function () {
+		provider = getTestObjectProvider();
+		// ODSP specific bug
+		if (provider.driver.type !== "odsp") {
+			this.skip();
+		}
+		configProvider.set("Fluid.ContainerRuntime.IdCompressorEnabled", true);
+		configProvider.set("Fluid.Runtime.UseShortIds", true);
```

Comment:
You only need this. Not the id compressor one.

Reply:
Updated